ui/src/main/resources/FileManagerCode/Macros.xml

```diff
@@ -418,14 +418,13 @@
       ## Add the file object.
       #set ($discard = $fileDoc.newObject('FileManagerCode.FileClass').set('description', "$!request.description"))
       ## Add the attachment.
-      ## FIXME: Get the file content as an input stream instead of a byte array, but for this we need to expose
-      ## getFileItemInputStream in the public API of the fileupload plugin (or use a script service).
-      #set ($bytes = $xwiki.fileupload.getFileItemData('filepath'))
-      #if (!$bytes)
-        ## Empty file (unfortunately getFileItemData returns null instead of an empty byte array).
-        #set ($bytes = [])
+      ## find the attachment among the fields of this form-submission
+      #set($fileuploadFiles = $xwiki.fileupload.getFileItems())
+      #set($inputstream = "")
+      #foreach($fileitem in $fileuploadFiles)
+        #if('filepath'==$fileitem.getFieldName()) #set($inputstream = $fileitem.getInputStream()) #end
```

Comment:
You should use camel case: ``fileUploadFiles``, ``inputStream``, ``fileItem``. Also, keeping the IF body on the same line makes it harder to read. I prefer to have it on separate lines, unless the IF is inside a literal string, which is not the case here. Finally, it would be nice to have a consistent code style. Check the style of the existing Velocity code.

Reply:
Thanks. Fixed in 2489c5abcdb3790e71b518d9f416e9229af78129.
paul